docs/quickstart/infrastructure.md

```diff
@@ -0,0 +1,127 @@
+<!--
+title: "Infrastructure monitoring with Netdata"
+sidebar_label: "Infrastructure monitoring"
+description: "Build a robust, infinitely-scalable infrastructure monitoring solution with Netdata. "
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/quickstart/infrastructure.md
+-->
+
+# Infrastructure monitoring with Netdata
+
+Together, the Netdata Agent and Netdata Cloud create a powerful, infinitely-scalable infrastructure monitoring solution.
+
+The Netdata Agent uses zero-configuration collectors to gather metrics from every application and container instantly.
+The distributed data model lets you monitor everything at a fraction of the cost compared to centralized metrics
```

Comment:
`The distributed data model lets you monitor everything at a fraction of the cost compared to centralized metrics` It might make sense to use a different term than cost as some might associate it to $$ and we don't charge. Perhaps incorporating faction of resources? Or something along those lines.